setup.cfg

```diff
@@ -109,3 +109,10 @@ line_length=88
 skip = ./.git
 ignore-words = dev/codespell/ignore-words.txt
 exclude-file = dev/codespell/exclude-file.txt
+
+[flake8]
+# https://flake8.pycqa.org/en/latest/user/error-codes.html
+# W503: line break before binary operator
+ignore = W503
+exclude = extern
+>>>>>>> Fix some Flake8 alerts
```

Comment:
Here is a left over from a merge conflict...please remove.

Reply:
Done, but I need to review the results of the merge one more time.